examples/binaries/new-meta/state-v3/build.rs

```diff
@@ -0,0 +1,21 @@
+// This file is part of Gear.
+
+// Copyright (C) 2021-2022 Gear Technologies Inc.
```

Comment:
2023

Reply:
and for others too

Reply:
Let's change licenses in separate PR all over the project